simpleline/event_loop/main_loop.py

```diff
@@ -196,12 +204,23 @@ def process_signals(self, return_after=None):
             if self._check_if_signal_processed(return_after, unique_id):
```

Comment:
`_check_if_signal_processed` doesn't need to check if `return_after` is None, no?

Reply:
Fixed.